### PR TITLE
data-migration: add dmctl command mode

### PR DESCRIPTION
**Pull request by 3pointer**
### What is changed, added or deleted? 
Add new feature (command mode) introduction about dm.

### What is the related PR or file link(s)? 
https://github.com/pingcap/dm/pull/287/files

### Which version does your change affect? 
N/A

### CODE REVIEW

**Comment by GregoryIan** [comment]
is this pr ready noew? @3pointer

**Comment by TomShawn** [comment]
@3pointer When the content is ready, please also update v3.1 as well, thanks~

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 1]
```diff
@@ -6,10 +6,11 @@ category: reference
 # 管理数据同步任务
 
 本文介绍了如何使用 [dmctl](/dev/reference/tools/data-migration/overview.md#dmctl) 组件来进行数据同步任务的管理和维护。对于用 DM-Ansible 部署的 DM 集群，dmctl 二进制文件路径为 `dm-ansible/dmctl`。
+dmctl 支持交互模式用于人工操作，同时也支持命令模式用于脚本。
```
Comment (csuzhangxc) [inline dev/reference/tools/data-migration/manage-tasks.md]: 这个前面，是不是要加个空行来开始一个新段落？

**Comment by GregoryIan** [comment]
@3pointer please take a time to address comments

File: dev/reference/tools/data-migration/manage-tasks.md (hunk 1 above)
Comment (3pointer) [reply dev/reference/tools/data-migration/manage-tasks.md]: Done

**Comment by csuzhangxc** [comment]
@TomShawn PTAL

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 2]
```diff
@@ -901,6 +903,50 @@ purge-relay -w "127.0.0.1:8262" --filename "mysql-bin.000003"
 >
 > 一般不需要使用此命令。仅当已确定 `task => DM-workers` 映射关系存在，但执行其它命令时仍提示必须刷新它时，你才需要使用此命令。
 
+## dmctl 基本用法--命令模式
+
+命令模式跟交互模式的区别是，执行时只需要在 dmctl 命令后紧跟任务操作执行，任务操作同交互模式参数一致。
+
+> **注意：**
+>
+> 1. 一次 dmctl 命令只能跟一个任务操作
```
Comment (TomShawn) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
> + 一条 dmctl 命令只能跟一个任务操作
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 3]
```diff
@@ -901,6 +903,50 @@ purge-relay -w "127.0.0.1:8262" --filename "mysql-bin.000003"
...
+>
```
Comment (TomShawn) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 4]
```diff
@@ -901,6 +903,50 @@ purge-relay -w "127.0.0.1:8262" --filename "mysql-bin.000003"
...
+> 2. 任务操作只能放在 dmctl 命令的最后
```
Comment (TomShawn) [inline dev/reference/tools/data-migration/manage-tasks.md]:
```suggestion
> + 任务操作只能放在 dmctl 命令的最后
```

File: dev/reference/tools/data-migration/manage-tasks.md [hunk 5]
```diff
@@ -901,6 +903,50 @@ purge-relay -w "127.0.0.1:8262" --filename "mysql-bin.000003"
...
+{{< copyable "shell-regular" >}}
+
+```bash
+./dmctl -master-addr 172.16.30.14:8261 start-task task.yaml
+./dmctl -master-addr 172.16.30.14:8261 stop-task task
+./dmctl -master-addr 172.16.30.14:8261 query-status
+```
+
+```
+Available Commands:
```
Comment (TomShawn) [inline dev/reference/tools/data-migration/manage-tasks.md]: What about taking the `Available Commands` out of the code block?

Comment (3pointer) [reply dev/reference/tools/data-migration/manage-tasks.md]: It's kind of output of dmctl, I think if fine put it here @csuzhangxc What do you think?

Comment (csuzhangxc) [reply dev/reference/tools/data-migration/manage-tasks.md]: Putting it in the code block is fine. (It's kind of output of dmctl)

File: dev/reference/tools/data-migration/manage-tasks.md (hunk 4 above)
Comment (3pointer) [reply dev/reference/tools/data-migration/manage-tasks.md]: seems strange, Why change `2` to `+`

Comment (TomShawn) [reply dev/reference/tools/data-migration/manage-tasks.md]: If there is a definite sequence for a list (one item must be listed only before or after another), such as procedures for a task, you can use an ordered list `1. 2. 3.`. If there is no such sequence and every item is just listed altogether, the unordered list must be used instead. By the way, in a note,  a blank line should not appear between items.

Comment (3pointer) [reply dev/reference/tools/data-migration/manage-tasks.md]: ok, seems also need change `1.` to `+`

Comment (TomShawn) [reply dev/reference/tools/data-migration/manage-tasks.md]: https://github.com/pingcap/docs-cn/pull/1901#discussion_r357467803

**Pull request merged**